lib/main_common.pm

```diff
@@ -1487,9 +1487,7 @@ sub load_extra_tests_textmode {
     }
     loadtest "console/cron";
     loadtest "console/syslog";
-    unless (is_sle '<15') {
-        loadtest "console/ntp_client";
-    }
+    loadtest "console/ntp_client" unless is_sle;
```

Comment:
I don't think we should keep it disabled for SLE, especially not without any comment. Pretty sure there are *some* expectations on the different versions of SLE. We should find out the specific expectations and work based on these.